src/common/meta/src/error.rs

```diff
@@ -70,6 +70,9 @@ pub enum Error {
         source: BoxedError,
         location: Location,
     },
+
+    #[snafu(display("Etcd txn error: {err_msg}"))]
+    EtcdTxnOpResponse { err_msg: String, location: Location },
```

Comment:
```suggestion
    #[snafu(display("Etcd txn error: {}, location: {}", err_msg , location))]
```

Reply:
Do we need to carry "location" in snafu display now? I see there are too many errors in current codebase that don't carry it.